javaparser-core/src/main/java/com/github/javaparser/JavaToken.java

```diff
@@ -139,10 +139,6 @@ public String toString() {
         return text;
     }
 
-    /**
-     * Check if the position is usable. Does not know what it is pointing at, so it can't check if the position is after
-     * the end of the source.
-     */
     public boolean valid() {
```

Comment:
Oh, this was a copy/paste error from `Range`, I think. I really should write the correct documentation.